main.go

```diff
@@ -21,7 +21,7 @@ var (
 )
 
 func init() {
-	//logrus.SetLevel(logrus.DebugLevel)
+	logrus.SetLevel(logrus.DebugLevel)
```

Comment:
You probably left this in my accident, but maybe you could _also_ add logic that will check an env var for debug. I think we use `RANCHER_DEBUG` but am not sure. Please verify

Reply:
cool, will use RANCHER_DEBUG